crates/resolver/src/async_resolver.rs

```diff
@@ -190,7 +190,7 @@ impl<R: RuntimeProvider> AsyncResolver<GenericConnection, GenericConnectionProvi
     }
 
     /// Flushes/Removes all entries from the cache
-    pub async fn clear_cache(&mut self) {
+    pub async fn clear_cache(&self) {
```

Comment:
So it looks like this doesn't need to be `async` either?

Reply:
It is true, it is not required. I marked it `async` to be consistent with the other methods.
Changing it back in the future should be fine if we need it, yes?

Reply:
No, we should just make it no-`async` now to avoid the overhead of the runtime machinery.

Reply:
Done :) waiting for CI to pass